app/graphql/types/query_type.rb

```diff
@@ -27,5 +30,13 @@ def all_event_records
     def event_record(id:)
       EventRecord.find(id)
     end
+
+    def all_news_items
+      NewsItem.all
+    end
+
+    def news_item(id:)
+      NewsItem.find(id)
+    end
```

Comment:
ich bleibe dabei, das ich mit dem Naming der `QueryTypes` nicht glücklich bin und habe mittlerweile auch einige BestPractices dazu gefunden.

- https://graphql.github.io/learn/thinking-in-graphs/
- https://mattboldt.com/2019/01/07/rails-and-graphql/
- https://developers.facebook.com/docs/graph-api/reference/

Reply:
Ok, dann ändere ich es.

Reply:
Im rails graphql tutorial, war es eben aber anders und davon bin ich ausgegangen

Reply:
Habe direkt ein issue dazu gemacht: #38